source/help/submit/index.md

```diff
@@ -50,6 +50,25 @@ We do not accept [dvi, PS, or PDF created
 from TeX/LaTeX source](../faq/whytex.md), and we
 do not accept scanned documents, regardless of format.
 
+
+<span id="newtex"></span>
+## (La)TeX processing changes &mdash; April 2024
+
+We will soon be rolling out changes to how arXiv process (La)TeX subbmissions. These changes shouldn't be noticable to most of our users. We will be retiring the arXiv-developed "AutoTeX' system that we've used for decades in favor of a simpler, more straight-foward process of convering (La)TeX submissions to PDFs..
+
+ 1. The AutoTeX would try different versions of TeX to see which one successfully builds a PDF. Going forward we will only use the version of TeX currently used by arXiv
+    - Our plan is that arXivs "current" version closely tracck the annual TeX Live releases. In the past, we've often gone a few years between TeX updates
+ 1. The AutoTeX system would attempt to detect image files that were not referenced in the (La)TeX source for the paper and append those images to the end of the PDF for the paper. Based on recent feedback, we find that most authors are surprised by this behvior, and we will thus be discontinuing this practice. If an author wants images at the end of their paper, just use the normal TeX procedures for this (see https://latex-tutorial.com/tutorials/figures/).
+  1. The AutoTeX system would pre-load the LaTeX hyperref package to LaTeX submissions that do not already include it. This package adds changes various references to active links in the PDF document (eg, clicking on a reference jumps to the entry in the bibliography section). We will no longer automatically try do add this package. (Note – before you add a \usepackage{hyperref} to your main TeX file, check to see if the template you're using already has a reference by checking the PDF for clickable links; most do).
+1. The AutoTeX system would look for any references in a paper that looked like an arXiv paper ID (such as 	arXiv:2402.08954, or 2402.08954), and turn the into a hyperlink to the paper on arXiv. 
+We won't do that anymore. Authors should just write out \hyperref{https://arxiv.org/abs/2402.08954}.
```

Comment:
write out \href{https://arxiv.org/abs/2402.08954}{2402.08954}. Please see the hyperref documentation for more options.